pubsublite-spark-sql-streaming/pom.xml

```diff
@@ -176,6 +181,36 @@
                     <include>com.google.common.**</include>
                   </includes>
                 </relocation>
+                <!-- Take special care of grpc-netty-shaded, it uses the package
+                     io.grpc.netty.shaded.io.grpc.netty, which will cause the
+                     ServicesResourceTransformer to replace both occurrences of io.grpc -->
+                <relocation>
+                  <pattern>io.grpc.netty.shaded</pattern>
+                  <shadedPattern>
+                    com.google.cloud.hadoop.repackaged.gcs.io.grpc.netty.shaded
```

Comment:
You need to change `com.google.cloud.hadoop.repackaged.gcs` here and below to `com.google.cloud.pubsublite.repackaged` or something like this so it will be unique to this project.